yellowbrick/classifier/__init__.py

```diff
@@ -23,8 +23,11 @@
 ## Hoist visualizers into the classifier namespace
 from ..base import ScoreVisualizer
 from .base import ClassificationScoreVisualizer
-from .class_balance import ClassBalance, ClassPredictionError
+from .class_prediction_error import ClassPredictionError, class_prediction_error
 from .classification_report import ClassificationReport, classification_report
 from .confusion_matrix import ConfusionMatrix, confusion_matrix
 from .rocauc import ROCAUC, roc_auc
 from .threshold import DiscriminationThreshold, discrimination_threshold
+
+## Import from target for backward compatibility and classifier association
+from ..target.class_balance import ClassBalance, class_balance
```

Comment:
Still importing class balance into the classification module for easy reference (though it does break backward compatibility)

Reply:
do we want to add a deprecation warning here or somewhere else?

Reply:
I'm not actually sure how to add the deprecation warning - the deprecated part is that the function signature has changed, but the function is still there. I suppose I could check the type of the input and raise an exception; I'll do that in a follow on pull request so it's easier to see. 

As for deprecating this import - I'm also not exactly sure how to do that; but I do like this particular visualizer living in "two" places - target and classifier; since it's a classification based target analysis. What do you think?